doc/API/centreon-api-v2.yaml

```diff
@@ -3604,6 +3672,68 @@ components:
             $ref: "#/components/schemas/Downtime.Host"
         acknowledgement:
           $ref: "#/components/schemas/Acknowledgement.Host"
+    Comment.Host:
+      type: object
+      required: [comment]
+      properties:
+        date:
+          type: string
+          format: date-time
+          nullable: true
+          description: "Date of the comment (ISO8601)"
+        comment:
+          type: string
+          description: "Comment to add to the monitored host"
+          example: "HOST is down because it has been powered off"
+    Comment.Service:
+      type: object
+      required: [comment]
+      properties:
+        date:
+          type: string
+          format: date-time
+          nullable: true
+          description: "Date of the comment (ISO8601)"
+        comment:
+          type: string
+          description: "Comment to add to the monitored service"
+          example: "SERVICE is critical because ntpd service if stopped"
+    Comment.Resources:
+      type: object
+      required: [resources]
+      properties:
+        resources:
+          type: array
+          items:
+            type: object
+            required: [comment,id]
+            properties:
+              type:
+                type: string
+                enum: [host, service]
+                description: "Type of resource"
+                example: "service"
+              id:
+                type: integer
```

Comment:
I think we should discuss about the standard to declare number,  some times we use `type: integer` and sometimes
```
type: number
format: integer
```
we should be more consistent on this point.

Reply:
Accoring to OpenAPI Documentation: https://swagger.io/docs/specification/data-models/data-types/#numbers the format 
```
type: number
format: integer
```

Doesn't seems correct